### PR TITLE
Made some changes in write-pages.md that were not done previously to include you's instead of we's

### PR DESCRIPTION
**Pull request by KiranAkadas**
## Description
Changed the file write-pages.md to replace we with you in required places

## Related Issues
Addresses #18284

### CODE REVIEW

File: docs/docs/write-pages.md [hunk 1]
```diff
@@ -12,7 +12,7 @@ title: Write Out Pages
 >
 > You can help by making a PR to [update this documentation](https://github.com/gatsbyjs/gatsby/issues/14228).
 
-This is one of the last bootstrap stages before we hand off to webpack to perform code optimization and code splitting. Webpack builds a web bundle. It has no knowledge of Gatsby's core code. Instead, it operates only on files in the `.cache` directory. It also doesn't have access to all the Redux information that was built up during bootstrap. So instead, we create dynamic JavaScript and JSON files that are dependent on by the webpack application in the `.cache` directory (see [Building the JavaScript App](/docs/production-app/)).
+This is one of the last bootstrap stages before you hand off to webpack to perform code optimization and code splitting. Webpack builds a web bundle. It has no knowledge of Gatsby's core code. Instead, it operates only on files in the `.cache` directory. It also doesn't have access to all the Redux information that was built up during bootstrap. So instead, create dynamic JavaScript and JSON files that are dependent on by the webpack application in the `.cache` directory (see [Building the JavaScript App](/docs/production-app/)).
```
Comment (marcysutton) [inline docs/docs/write-pages.md]: The subject here is Gatsby, not the reader: see the Style Guide for more info: https://www.gatsbyjs.org/contributing/gatsby-style-guide/#use-you-as-the-pronoun

That would be true for the rest of this doc, and should be reworded accordingly.

**Comment by LekoArts** [comment]
@KiranAkadas Can you make the requested changes?

**Comment by KiranAkadas** [comment]
Yes! Onto it
Sorry, was a bit engaged with work.

**Comment by KiranAkadas** [comment]
Hello,
Please check the updated file and let me know if any changes are to be made.

**Comment by KiranAkadas** [comment]
Hello
Can anyone let me know why the tests here are failing for my commit?

**Comment by LekoArts** [comment]
Closed in favor of https://github.com/gatsbyjs/gatsby/pull/19843

**Pull request closed without merging**